### ISSUE TITLE
Train FlatModel and LCPNModel on Zooplankton images

### ISSUE THREAD

**Issue by EthanSansom** [EthanSansom/zooplankton#24]
### Tasks

- [x] Ensure hierarchy is correct
- [x] Ensure configuration is correct (e.g. learning rate, test-train-valid split)
- [x] Train FlatModel (backbone: Resnet18, pre-trained)
- [x] Train LCPNModel (backbone: Resnet18, pre-trained)
- [x] Train LCPNModel w/ FlatModel backbone (reduced learning rate)
